tbx/static_src/sass/components/_work-hero.scss

```diff
@@ -0,0 +1,56 @@
+@use "config" as *;
+
+.work-hero {
+    @include media-query(large) {
+        align-items: end;
+    }
+
+    &__left {
+        grid-column: 2 / span 4;
+
+        @include media-query(large) {
+            grid-column: 2 / span 8;
+        }
+    }
+
+    &__right {
+        grid-column: 2 / span 4;
+
+        @include media-query(large) {
+            grid-column: 11 / span 3;
+            grid-row: 1 / span 2;
+            margin-bottom: 7px; // aligns better with the bottom of the intro, accounting for line height
+        }
+    }
+
+    &__logo {
+        margin-bottom: $spacer-mini;
+        filter: grayscale(1) invert(1) brightness(1.5);
+
+        @include media-query(large) {
+            margin-bottom: $spacer-mini-plus;
+        }
+
+        @include high-contrast-light-mode() {
+            filter: none;
+        }
+    }
+
+    &__client {
```

Comment:
This looks to be white in the designs

Reply:
I tried making it white but them some logos became a blank square as they didn't have a transparent element. I'd like to review these and get Ben and Jeth's input. Do you think you could chat to ben about it when you see him tomorrow?